framework/test/unit/modules/nft/commands/transfer_cross_chain.spec.ts

```diff
@@ -0,0 +1,520 @@
+		});
+
+		interoperabilityMethod = {
+			send: jest.fn().mockResolvedValue(Promise.resolve()),
+			error: jest.fn().mockResolvedValue(Promise.resolve()),
+			terminateChain: jest.fn().mockResolvedValue(Promise.resolve()),
+			getMessageFeeTokenID: jest.fn().mockResolvedValue(Promise.resolve(messageFeeTokenID)),
+		};
+
+		internalMethod.init({
+			ownChainID,
+		});
+
+		internalMethod.addDependencies(nftMethod, interoperabilityMethod);
+
+		command.init({ nftMethod, tokenMethod, interoperabilityMethod, internalMethod });
+
+		existingNFT = {
+			owner,
+			nftID: Buffer.concat([ownChainID, utils.getRandomBytes(LENGTH_NFT_ID - LENGTH_CHAIN_ID)]),
+		};
+
+		lockedExistingNFT = {
+			owner,
+			nftID: Buffer.concat([ownChainID, utils.getRandomBytes(LENGTH_NFT_ID - LENGTH_CHAIN_ID)]),
+		};
+
+		escrowedNFT = {
+			owner: utils.getRandomBytes(LENGTH_CHAIN_ID),
+			nftID: utils.getRandomBytes(LENGTH_NFT_ID),
+		};
+
+		await nftStore.save(methodContext, existingNFT.nftID, {
+			owner: existingNFT.owner,
+			attributesArray: [],
+		});
+
+		await userStore.set(methodContext, userStore.getKey(existingNFT.owner, existingNFT.nftID), {
+			lockingModule: NFT_NOT_LOCKED,
+		});
+
+		await module.stores.get(NFTStore).save(methodContext, lockedExistingNFT.nftID, {
+			owner: lockedExistingNFT.owner,
+			attributesArray: [],
+		});
+
+		await userStore.set(
+			methodContext,
+			userStore.getKey(lockedExistingNFT.owner, lockedExistingNFT.nftID),
+			{
+				lockingModule: 'token',
+			},
+		);
+
+		await module.stores.get(NFTStore).save(methodContext, escrowedNFT.nftID, {
+			owner: escrowedNFT.owner,
+			attributesArray: [],
+		});
+
+		await userStore.set(methodContext, userStore.getKey(escrowedNFT.owner, escrowedNFT.nftID), {
+			lockingModule: NFT_NOT_LOCKED,
+		});
+
+		await tokenUserStore.set(methodContext, tokenUserStore.getKey(owner, messageFeeTokenID), {
+			availableBalance,
+			lockedBalances: [],
+		});
+	});
+
+	describe('verify', () => {
+		// let existingNFT: { nftID: any; owner: any };
+		// let lockedExistingNFT: { nftID: any; owner: any };
+		// let escrowedNFT: { nftID: any; owner: any };
+
+		// beforeEach(async () => {
+		// 	existingNFT = {
+		// 		owner,
+		// 		nftID: Buffer.concat([ownChainID, utils.getRandomBytes(LENGTH_NFT_ID - LENGTH_CHAIN_ID)]),
+		// 	};
+
+		// 	lockedExistingNFT = {
+		// 		owner,
+		// 		nftID: Buffer.concat([ownChainID, utils.getRandomBytes(LENGTH_NFT_ID - LENGTH_CHAIN_ID)]),
+		// 	};
+
+		// 	escrowedNFT = {
+		// 		owner: utils.getRandomBytes(LENGTH_CHAIN_ID),
+		// 		nftID: utils.getRandomBytes(LENGTH_NFT_ID),
+		// 	};
+
+		// 	await nftStore.save(methodContext, existingNFT.nftID, {
+		// 		owner: existingNFT.owner,
+		// 		attributesArray: [],
+		// 	});
+
+		// 	await userStore.set(methodContext, userStore.getKey(existingNFT.owner, existingNFT.nftID), {
+		// 		lockingModule: NFT_NOT_LOCKED,
+		// 	});
+
+		// 	await module.stores.get(NFTStore).save(methodContext, lockedExistingNFT.nftID, {
```

Comment:
please remove unused comments

Reply:
Thanks for pointing this, updated [here](https://github.com/LiskHQ/lisk-sdk/pull/8560/commits/ba4dadfdadc7567426e7f9f9c2b4b54cc8e79415).